Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -647,6 +667,25 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output.shape, expected)
     }
 
+    func testFlattenGradient() {
+        let layer = Flatten<Float>()
+        let input = Tensor(shape: [1, 4, 4], scalars: (0..<16).map(Float.init))
+        let computedGradient = gradient(at: input, layer) { $1($0).sum() }
+        // The expected value of the gradient was computed using the following Python code:
+        // ```
+        //   Flatten = tf.keras.layers.Flatten()
+        //   with tf.GradientTape() as t:
+        //     t.watch(x)
+        //     y = tf.math.reduce_sum(Flatten(x)
```

Comment:
```suggestion
        //     y = tf.math.reduce_sum(flatten(x))
```
Add a missing parenthesis.